e2e/support/helpers/e2e-misc-helpers.js

```diff
@@ -237,14 +252,6 @@ function visitDashboardById(dashboard_id, config) {
   });
 }
 
-/**
- * Visit a dashboard by using its previously saved dashboard id alias.
- * @param {string} alias
- */
-function visitDashboardByAlias(alias, config) {
```

Comment:
I decided to inline this function, because:
- it's 1 LOC
- it's used only in 1 place
- there would be a 1 function like that for: dashboard, question, model (in the future), metric (in the future), etc.